### ISSUE TITLE
Bug/1.2.0 - 32 bits build failed - most OpenWrt build fails

### ISSUE THREAD

**Issue by erdoukki** [crowdsecurity/crowdsec#979]
https://github.com/openwrt/packages/pull/16244/checks

```
Building targets
github.com/crowdsecurity/crowdsec/pkg/csplugin
# github.com/mattn/go-sqlite3
sqlite3-binding.c: In function 'sqlite3SelectNew':
sqlite3-binding.c:128049:10: warning: function may return address of local variable [-Wreturn-local-addr]
128049 |   return pNew;
       |          ^~~~
sqlite3-binding.c:128009:10: note: declared here
128009 |   Select standin;
       |          ^~~~~~~
# github.com/crowdsecurity/crowdsec/pkg/csplugin
../../pkg/csplugin/broker.go:403:20: constant 4294967295 overflows int
../../pkg/csplugin/broker.go:410:20: constant 4294967295 overflows int

make[2]: *** [Makefile:158: /home/build/openwrt/build_dir/target-mips_24kc_musl/crowdsec-1.2.0/.built] Error 2
make[2]: Leaving directory '/home/runner/work/packages/packages/net/crowdsec'
make[1]: *** [package/Makefile:116: package/feeds/packages_ci/crowdsec/compile] Error 1
time: package/feeds/packages_ci/crowdsec/compile#4.53#1.34#4.85
make: *** [/home/build/openwrt/include/toplevel.mk:223: package/crowdsec/compile] Error 2
    ERROR: package/feeds/packages_ci/crowdsec failed to build.
make[1]: Leaving directory '/home/build/openwrt'
Error: Process completed with exit code 2.
```

**Comment by erdoukki** [comment]
https://github.com/openwrt/packages/pull/16244

**Comment by blotus** [comment]
Hello,

This looks like it happen only when trying to build a 32 bits version of crowdsec, we'll try to reproduce the issue on our side.

**Comment by erdoukki** [comment]
Find this ; which may help ; https://github.com/moby/moby/issues/15038
`Use math.MaxInt32 instead of math.MaxUint32`
403:
`	if uid < 0 && uid > math.MaxUint32 {`
410:
`	if gid < 0 && gid > math.MaxUint32 {`

Will also give a try...

**Comment by erdoukki** [comment]
Fixed, all builds now correct on OpenWrt tests CI : https://github.com/openwrt/packages/actions/runs/1252838765
patch in PR : https://github.com/crowdsecurity/crowdsec/pull/980

**Comment by erdoukki** [comment]
> # github.com/crowdsecurity/crowdsec/pkg/csplugin
> ../../pkg/csplugin/broker.go:403:20: constant 4294967295 overflows int
> ../../pkg/csplugin/broker.go:410:20: constant 4294967295 overflows int

Same issue here: 
https://github.com/moby/moby/issues/15038

I propose same solution:
https://github.com/moby/moby/pull/15051

**Comment by erdoukki** [comment]
May be another resolution:
https://stackoverflow.com/questions/36231049/why-does-printing-out-max-int-cause-compile-error-in-golang

`uint32(math.MaxUint32)`

**Comment by blotus** [comment]
Hello @erdoukki,

I've just merged #980, i'll close the issue, do not hesitate to open another one if the error persists.
